_startc.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Entrypoint
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Hubert Buczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "config.h"
+#include "hal.h"
+
+
+extern void (*__init_array_start [])(void);
+extern void (*__init_array_end [])(void);
+
+extern void (*__fini_array_start [])(void);
+extern void (*__fini_array_end [])(void);
+
+extern void plo_init(void);
+
+
+void _startc(int argc, char **argv, char **env)
+{
+	size_t i, size;
+
+	/* Clear the .bss section */
+	hal_memset(plo_bss, 0, (addr_t)_end - (addr_t)plo_bss);
```

Comment:
`_plo_bss` not  defined in `TARGET=armv7a9-zynq7000`

Reply:
Fixed